### DISCUSSION TITLE
TypeError: _ReanimatedModule.default.configureProps is not a function

### DISCUSSION THREAD

**Discussion by nialldbarber** [software-mansion/react-native-reanimated#3309] [Q&A]
I am reluctant to create an issue for this as I am in the process of updating our app from V1 to V2, which also includes various updates to React Native `0.67`, React Navigation 6 etc, so there could be issues there. 

But we only use reanimated in 2 files throughout the app and it's based on the old V1 API. But reanimated is backwards compatible, so this _shouldn't_ be an issue as far as I'm aware. 

However, every time I try to build the project, I encounter this error:

```
TypeError: _ReanimatedModule.default.configureProps is not a function
```

I can't find a great deal in terms of other people getting this error so it's hard to debug. We are not using expo, but even reinstalling packages, resetting the cache, same with the pods as well. 

Anyone else had this error before or something similar? 

My list of packages include: 

```
"react": "17.0.2",
"react-native": "0.67.4",
"react-native-reanimated": "2.8.0",
```
<img width="375" alt="Screenshot 2022-06-23 at 15 27 04" src="https://user-images.githubusercontent.com/20539827/175323506-1da229a9-1ec8-4028-a809-3f943d07a863.png">

**Comment by tomekzaw** [comment]
Hey, can you please paste the output of `npm ls react-native-reanimated`? There might be two different versions of Reanimated installed simultaneously (if some other dependency has a hard dependency on a specific version of Reanimated).

**Reply by Bryan47749** [reply]
Hi tomekzaw, I have same problem. In my case, react-native-reanimated depends on react-native-neu-element
![Captura](https://user-images.githubusercontent.com/41696489/206574794-45cb52d6-3629-4d88-91d7-cd4e07c47e37.PNG)
